src/Entity/ItemId.php

```diff
@@ -24,7 +24,7 @@ class ItemId extends EntityId implements Int32EntityId {
 	 */
 	public function __construct( $idSerialization ) {
 		$this->assertValidIdFormat( $idSerialization );
-		$this->serialization = strtoupper( $idSerialization );
+		parent::__construct( parent::normalizeIdSerialization( $idSerialization ) );
```

Comment:
What motivated using a constructor in the parent class? Definitely not worth it IMO.